espnet/nets/chainer_backend/e2e_asr.py

```diff
@@ -70,8 +70,9 @@ def __init__(self, idim, odim, args):
 
         self.acc = None
         self.loss = None
+        self.flag_return = flag_return
 
-    def __call__(self, xs, ilens, ys):
+    def __call__(self, xs, ilens, ys, _old=False):
```

Comment:
Is `_old` used?

Reply:
Sorry, i did not realize that I forgot to delete that variable. I was testing different ways to return the loss and acc and this was one of those. I just updated the PR without this var.